torch/distributed/_shard/metadata.py

```diff
@@ -33,6 +33,18 @@ def __init__(
         shard_sizes: List[int],
         placement: Optional[Union[str, _remote_device]] = None
     ):
+        """
```

Comment:
Please remove the new dosctrings and only fix the errors listed in the issue. I think the issues clearly asks to skip the "Missing docstring" errors.